templates/whats-coming-up.html

```diff
@@ -0,0 +1,65 @@
+{% block upcoming_events %}
+
+<div class="p-strip--light">
+  <div class="row">
+    <div class="col-12">
+      <h5>Upcoming events and webinars</h5>
+    </div>
+  </div>
+
+  {%- for post in upcoming_events %}
+  {% if loop.index0 % 3 == 0 %}
+  <div class="row u-equal-height u-clearfix">
+  {% endif %}
+  <div class="col-4 p-card--post">
+    <header class="p-card__header--{{ post.group.slug }}">
+      <h5 class="p-muted-heading">{{ post.group.name }}</h5>
+    </header>
+    <div class="p-card__content">
+      {% if post.featuredmedia %}
+      <div class="u-crop--16-9">
+        <a href="{{post.link}}">
+          <img src="{{post.featuredmedia.source_url}}" alt="{{post.featuredmedia.alt_text}}" />
+        </a>
+      </div>
+      {% endif %}
+      <h3 class="p-heading--four"><a href="{{ post.link }}">{{ post.title.rendered | safe }}</a></h3>
+      {% if not post.featuredmedia %}
+      <p class="u-no-padding--bottom">{{ post.summary | striptags | urlize(30, true) }}</p>
+      {% endif %}
+      <ul class="p-list">
+        {% if post.start_date and post.end_date %}
+        <li class="p-list__item">
+          <i class="p-icon--calendar"></i>&nbsp;
+          {{ post.start_date }} &ndash; {{ post.end_date }}
```

Comment:
Webinars have the same start and end date, to avoid duplication, what about: 
`{{ post.start_date }}{% ifnotequal post.start_date post.end_date %} &ndash; {{ post.end_date }}{% endifnotequal %}`

Reply:
@caldav Thanks. I've updated the template.